### DISCUSSION TITLE
对每个模型厂商都加入模型控制列表

### DISCUSSION THREAD

**Discussion by RubuJam** [lobehub/lobehub#2103] [General]
目前已经对OpenAi和Azure OpenAI分别添加了模型控制支持，即`OPENAI_MODEL_LIST `和`AZURE_MODEL_LIST`

在`v 1.0`版本中会彻底去除`CUSTOM_MODELS`这一环境变量，尽管可以通过修改源代码去启用/禁用各个模型，但依然有些烦琐，没有通过环境变量设置简便

**Comment by arvinxx** [comment]
@RubuJam 不要动不动提所有。你有需求的按需求提，讲明要加的原因。而且现在也支持了 `OPENROUTER_MODEL_LIST`、`OLLAMA_MODEL_LIST`、`TOGETHERAI_MODEL_LIST` ，我觉得已经基本上满足95%的需求了。

如果你有能力动手，也欢迎 PR 。参考这里来就好：https://github.com/lobehub/lobe-chat/blob/main/src/app/api/config/route.ts#L88-L105

**Comment by RubuJam** [comment]
已通过#2574 #2634 解决
